spec/features/results_download_spec.rb

```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
```

Comment:
As this scope describes "csv downloading process", I'd not check if there is this selector but try to click it like in the first spec.